### PR TITLE
Add example of `this` for #43

### PR DESCRIPTION
**Pull request by jvican**

### CODE REVIEW

**Comment by jvican** [comment]
I have no idea why that title has been chosen. It seems to be a bug of `hub` or something.

File: core/src/test/scala/scala/spores/neg/sip21/AkkaLikeFutureNeg.scala [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      }
+
+      trait ActorReceiver[T] {
+        def receive(r: Request[T]): Future[NullarySpore[String]]
+      }
+
+      import scala.concurrent.ExecutionContext.Implicits.global
+
+      object ConcreteReceiver extends ActorReceiver[String] {
+        val sender: ActorSender[String] = ConcreteSender
```
Comment (phaller) [inline core/src/test/scala/scala/spores/neg/sip21/AkkaLikeFutureNeg.scala]: In Akka, `sender` (actually, `sender()`) is a method. Perhaps you could simulate that.

File: core/src/test/scala/scala/spores/neg/sip21/AkkaLikeFutureNeg.scala [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+      case class Request[T](value: T)
+      case class Response[T](value: T)
+
+      trait ActorSender[T] {
+        def send(r: Response[T]): Future[T]
+      }
+
+      object ConcreteSender extends ActorSender[String] {
+        def send[T](r: Response[T]): Future[T] = ???
```
Comment (phaller) [inline core/src/test/scala/scala/spores/neg/sip21/AkkaLikeFutureNeg.scala]: Returning a future here is a bit surprising, and unlike Akka, so people reading the example might get tripped up by this. ;-)

File: core/src/test/scala/scala/spores/neg/sip21/AkkaLikeFutureNeg.scala (hunk 1 above)
Comment (jvican) [inline core/src/test/scala/scala/spores/neg/sip21/AkkaLikeFutureNeg.scala]: Perfect, I literally forgot everything about the Akka API so this PR is just a PoC. But I'll address those issues :+1:

File: core/src/test/scala/scala/spores/neg/sip21/AkkaLikeFutureNeg.scala (hunk 2 above)
Comment (jvican) [inline core/src/test/scala/scala/spores/neg/sip21/AkkaLikeFutureNeg.scala]: Okey, agreed, thanks for the tip!

**Comment by jvican** [comment]
This guy depends on #46! Do not merge until that one is merged, since I'll need to rebase. @phaller FYI, I've copied the public Akka API so that tests are more "realistic", more info in #43.

**Pull request closed without merging**